openlibrary/templates/about/team_member.html

```diff
@@ -0,0 +1,59 @@
+<link
+	href="$static_url('build/page-teamMember.css')"
+	rel="stylesheet"
+	type="text/css" />
+$var title: $_("Team Member's Name")
+
+<h2 class="teamPageTitle">The Open Library Team</h2>
+<h1 class="teamMember__name">Arwen</h1>
+<h3 class="teamMember__title">Lead Engineer</h3>
+<div class="container">
+	<div class="teamMember__sideBar__container">
+		<div class="sideBar">
+			<div class="sideBar__photoContainer">
+				<img
+					src="../../../static/images/team-arwen.jpg"
```

Comment:
As a next step, let's remove the place holder images and just start w/ the core team (not to play favorites, but because it's hopefully a very reachable goal).

@cdrini, @jimchamp, @seabelis, @mekarpeles, @cclauss, @scottbarnes.

Reply:
Sounds good! I'll work on that this week and I'll reach out if I have any questions.